patches/base-threading-thread_restrictions.h.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/base/threading/thread_restrictions.h b/base/threading/thread_restrictions.h
-index 8eb5907fd9278f8f246fcbd83a354ebe757bdc1e..3f29a1a3c349be38d635890f912218ef89a6ffca 100644
+index e029700325e792e7a840ed94f25e7a49b86d0046..f9d878877802a00171552a7a5b5e159df3198f67 100644
 --- a/base/threading/thread_restrictions.h
 +++ b/base/threading/thread_restrictions.h
-@@ -729,6 +729,7 @@ class BASE_EXPORT ScopedAllowBaseSyncPrimitives {
+@@ -565,6 +565,7 @@ class BASE_EXPORT ScopedAllowBlocking {
+   ScopedAllowBlocking(const ScopedAllowBlocking&) = delete;
+   ScopedAllowBlocking& operator=(const ScopedAllowBlocking&) = delete;
+ 
++  BRAVE_SCOPED_ALLOW_BLOCKING_H
```

Comment:
I don't understand what's going on here, why are we making changes to `ScopedAllowBlocking`?